src/main/java/seedu/recruittrackpro/model/ModelManager.java

```diff
@@ -128,6 +128,16 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void switchPersonListSorting() {
+        recruitTrackPro.getUniquePersonList().switchSorting();
+    }
+
+    @Override
+    public boolean isAscending() {
+        return recruitTrackPro.getUniquePersonList().isAscending();
+    }
+
```

Comment:
This violates Law of Demeter, principle, i.e. this method tells `RecruitTrackPro` object to give `ModelManager` its `UniquePersonList` object, then tell that object to do something.

Consider shifting the responsibility upwards, to RecruitTrackPro:

```suggestion
    @Override
    public void togglePersonListSortOrder() {
        recruitTrackPro.togglePersonListSorting();
    }

    @Override
    public boolean isPersonListSortedAscending() {
        return recruitTrackPro.isPersonListSortedAscending();
    }